test/Sentry.Tests/Platforms/iOS/NativeSerializationTests.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Foundation;
+using Sentry.Cocoa.Extensions;
+using Xunit;
+
+namespace Sentry.Tests.Platforms.iOS;
+
+public class NativeSerializationTests
```

Comment:
```suggestion
public class CocoaExtensionsTests
```

Just to keep things tidy, we usually have the name of the test classes match the class exposing the functionality (only with a `Tests` suffix). That way it's easy to determine whether any tests exist for a particular class/method or not.